### DISCUSSION TITLE
Change appearance of buttons?

### DISCUSSION THREAD

**Discussion by Jajjaman** [dbuezas/lovelace-plotly-graph-card#327] [Q&A] [answered]
Where is the CSS (or how) to change the appearance of the buttons?

**Comment by dbuezas** [comment]
Which buttons specifically?

**Comment by Jajjaman** [comment]
The buttons where one can choose to show differnet timeframes. Like under the graph here
![CleanShot 2023-11-18 at 19 48 43@2x](https://github.com/dbuezas/lovelace-plotly-graph-card/assets/143397143/d7dd552e-0713-4340-a11e-18e495f54c74)

**Reply by dbuezas** [reply]
see here https://plotly.com/javascript/custom-buttons/#style-the-buttons

**Comment by Jajjaman** [comment, marked as answer]
Yes. Thank you.
